Sources/Util/Util.swift

```diff
@@ -22,6 +22,17 @@ public func isAnImage(_ file: String) -> Bool {
            file.hasSuffix("png")
 }
 
+public func getRange(of rawRange: String, length contentLength: Int) -> Range<Int> {
+  let splitRange = rawRange.components(separatedBy: "-")
+
+  let rangeEnd = Int(splitRange.last!) ?? contentLength - 1
+
+  let rangeStart = Int(splitRange.first!) ?? contentLength - rangeEnd
```

Comment:
With `components`, the method will return the original string inside an array if the string doesn't contain the passed argument. So is it okay to force-unwrap in this case since I know that `splitRange` will always have a first and last element?